interfaces/builtin/screen_inhibit_control.go

```diff
@@ -26,6 +26,7 @@ import (
 const screenInhibitControlConnectedPlugAppArmor = `
 # Description: Can inhibit and uninhibit screen savers in desktop sessions.
 #include <abstractions/dbus-session-strict>
+#include <abstractions/dbus-strict>
```

Comment:
Hi! The Unity screen API, which is already allowed in the the screen_inhibit_control.go file, needs access to the system bus:

```
# unity screen API
dbus (send)
    bus=system
    interface="org.freedesktop.DBus.Introspectable"
    path="/com/canonical/Unity/Screen"
    member="Introspect"
    peer=(label=unconfined),
dbus (send)
    bus=system
    interface="com.canonical.Unity.Screen"
    path="/com/canonical/Unity/Screen"
    member={keepDisplayOn,removeDisplayOnRequest}
    peer=(label=unconfined),
```